posthog/warehouse/models/datawarehouse_saved_query_query_type.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class DataWarehouseSavedQueryQueryType(models.TextChoices):
+    """The type of query to generate"""
+
+    REVENUE_ANALYTICS_CHARGE = ("revenue_analytics_charge", "Revenue Analytics Charge")
```

Comment:
I need this for snapshots too. Can you add a "view" type that would be the normal/default view

Reply:
Hmm, what do you mean? It will imply that we should use whatever's inside `query` as is, rather than computing the query programmatically? I'm assuming that's the same as `None`, but it might make sense to add it here and make this a non-nullable field.